DogeNet.BLL/Features/Conversations/CreateConversation/CreateConversationHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿// <copyright file="CreateConversationHandler.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Conversations.CreateConversation
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using DogeNet.BLL.Services.Interfaces;
+    using DogeNet.DAL;
+    using DogeNet.DAL.Models;
+    using MediatR;
+    using Microsoft.AspNetCore.Identity;
+
+    public class CreateConversationHandler : IRequestHandler<CreateConversationCommand, Conversation>
+    {
+        private readonly DBContext context;
+
+        private readonly IMapper mapper;
+
+        public CreateConversationHandler(DBContext context, IMapper mapper)
+        {
+            this.context = context;
+            this.mapper = mapper;
+        }
+
+        public async Task<Conversation> Handle(CreateConversationCommand request, CancellationToken cancellationToken)
```

Comment:
We need to have a separate model for returning to UI. We shouldn't return classes from DAL.

Reply:
done